layouts/partials/stylesheets.html

```diff
@@ -3,3 +3,9 @@
 
 {{ $noscript_css := resources.Get "css/noscript.css" | minify | fingerprint "sha512" }}
 <noscript><link rel="stylesheet" href="{{ $noscript_css.RelPermalink }}" integrity="{{ $noscript_css.Data.Integrity }}"></noscript>
+
+{{ $custom_css := .Site.Params.custom_css | default "" }}
+{{ $custom_css_loc := resources.Get $custom_css }}
+{{ if $custom_css }}
```

Comment:
I would use `with` instead of a temporary variable which avoids having to make an empty string the default and having two (possibly confusing) variables.
Also, I would minify the stylesheet, add a fingerprint to it (so changes to the file are picked up by browsers even when a long caching period is set) and add SRI (which, admittedly, is somewhat pointless for files on the same host, but definitely does not hurt to add).
```suggestion
{{ with .Site.Params.custom_css }}
{{ $custom_css := resources.Get . | minify | fingerprint "sha512" }}
<link rel="stylesheet" href="{{ $custom_css.RelPermalink }}" integrity="{{ $custom_css.Data.Integrity }}">
{{ end }}
```